### ISSUE TITLE
ImportError in Black linter and code formatter

### ISSUE THREAD

**Issue by khurrammaqbool** [Clinical-Genomics/BALSAMIC#900]
**Describe the bug**
 Black linter and code formatter failed in github actions with the following error:

```
Run # Exists since using github.action_path + path to main script doesn't work because bash
Traceback (most recent call last):
  File "/home/runner/work/_actions/psf/black/stable/.black-env/bin/black", line 8, in <module>
    sys.exit(patched_main())
  File "/home/runner/work/_actions/psf/black/stable/.black-env/lib/python3.7/site-packages/black/__init__.py", line 1129, in patched_main
    patch_click()
  File "/home/runner/work/_actions/psf/black/stable/.black-env/lib/python3.7/site-packages/black/__init__.py", line 1115, in patch_click
    from click import _unicodefun  # type: ignore
ImportError: cannot import name '_unicodefun' from 'click' (/home/runner/work/_actions/psf/black/stable/.black-env/lib/python3.7/site-packages/click/__init__.py)
Error: Process completed with exit code 1.
```

**Version (please complete the following information):**
`balsamic, version 8.2.8`

**Comment by khurrammaqbool** [comment]
https://github.com/psf/black/issues/2964

**Comment by khurrammaqbool** [comment]
fixed by updating from version `21.7b0` to `22.3.0`
